.github/workflows/build.yml

```diff
@@ -46,9 +45,25 @@ jobs:
           java-version: ${{matrix.java.version}}
           cache: 'maven'
 
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.5
+        with:
+          maven-version: 3.8.6
```

Comment:
Why this is necessary?

Reply:
Hi @gastaldi, what's up?
Well, when I executed locally with (nektos/act) I needed to set this step. If no configuration is necessary to run mvn here, I can remove.

Reply:
The `actions/setup-java` already setups Maven, therefore I don't think that's necessary. The only reason I can think of to keep it is if we want to fix the Maven version  but then using the Maven Wrapper (mvnw) is better in this case.